### PR TITLE
add default lang to the config value

### LINKED ISSUE #12: Extend the config object
Extend the configuration object with the default lang: `{defaultLang: string}`
then we will use the default lang and initial the `lang$` observable in `translateService`

### PR DESCRIPTION
**Pull request by itayod**
close #12

### CODE REVIEW

File: projects/transloco/src/lib/transloco.service.ts [hunk 1]
```diff
@@ -1,17 +1,26 @@
...
+
+  get lang$() {
+    return this.lang.asObservable().pipe(distinctUntilChanged());
+  }
   cache = new Map<string, Observable<{ [key: string]: any }>>();
 
-  constructor(@Inject(TRANSLOCO_LOADER) private loader: Loader) {}
+  constructor(
+    @Inject(TRANSLOCO_LOADER) private loader: Loader,
+    @Optional() @Inject(TRANSLOCO_CONFIG) config: TranslocoConfig = defaults
```
Comment (NetanelBasal) [inline projects/transloco/src/lib/transloco.service.ts]: The defaults assignment will not work here, as the value is `null`.

Comment (NetanelBasal) [inline projects/transloco/src/lib/transloco.service.ts]: You should merge the config with the defaults. Check out the directive.

Comment (itayod) [reply projects/transloco/src/lib/transloco.service.ts]: good catch, I have updated the pr.

File: projects/transloco/src/lib/transloco.service.ts [hunk 2]
```diff
@@ -1,17 +1,26 @@
...
 
 @Injectable({
   providedIn: 'root'
 })
 export class TranslocoService {
-  private lang = new BehaviorSubject<string>('en');
-  lang$ = this.lang.asObservable().pipe(distinctUntilChanged());
+  private lang: BehaviorSubject<string>;
+
+  get lang$() {
```
Comment (NetanelBasal) [inline projects/transloco/src/lib/transloco.service.ts]: This will return a new Observable each time. It's better to use a property.

**Comment by itayod** [comment]
@NetanelBasal I have updated the pr with some changes because of conflicts... could you rereview it?

**Pull request merged**